sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -322,3 +322,8 @@ SELECT INTERVAL '153722867280' MINUTE;
 SELECT INTERVAL '-153722867280' MINUTE;
 SELECT INTERVAL '54.775807' SECOND;
 SELECT INTERVAL '-54.775807' SECOND;
+
+SELECT INTERVAL '1' DAY > INTERVAL '1' HOUR;
+SELECT INTERVAL '1' YEAR = INTERVAL '1' MONTH;
+SELECT INTERVAL '1-1' YEAR TO MONTH = INTERVAL '13' MONTH;
+SELECT INTERVAL '13' MONTH > INTERVAL '1' YEAR;
```

Comment:
Could you add more tests:
- At least, every supported ANSI interval type should be tested
- More compare ops
- Negative, 0, positive intervals
- Negative tests
- Comparison of year-month with day-time intervals (should fail).

Reply:
Update a lot test, should be enough